### DISCUSSION TITLE
Trouble reading *one* password

### DISCUSSION THREAD

**Discussion by hcgrove** [zeapo/Android-Password-Store#2409] [Q&A] [answered]
I have a repository, that I can reliably sync between my computer and my phone.
On my computer I can't see a difference between `a.gpg` and `b.gpg`, `gpg` says both are encrypted with the same two keys (one is the one I use on the computer, the other is the one I - want to - use on the phone), and `pass show a` and `pass show b` both show the passwords, usernames and extra content.

But when I sync the repo to my phone, (android) "Password Store" (APS) will only show the data (password+username+extra content) in one of them (a), but says (android says I can't take a screenshot because the page contains personal information) "This key is not available. To use it, you must import it as one of your own!" - above that there is text explaining that it needs access to the key and some information about the key which does not match any of the keys the information is encrypted with (the creation date it shows is from before I started playing with APS).

How do I go about finding out what APS is trying? Or more importantly: Can I be sure that it's just this one password (that won't be needed on the phone), that was just the first (I think so at least) I added to that repo, that is broken when I copy more encrypted files (from another repo) to this?

**Comment by hcgrove** [comment, marked as answer]
After I re-initialised the repository (for adding the key used on my new phone), the problem has gone away, I can see both passwords on the computer and both phones.

If anyone has an idea what was wrong, I would like to hear (it might improve my understanding of how all components in this works, though I thought I did) - but as I don't have the problem any longer I can't try anything to help locate the problem.

**Reply by msfjarvis** [reply]
With the little information we have it is hard to guess what the problem was, if it resurfaces I can try to debug it.
